daemon/top_windows.go

```diff
@@ -0,0 +1,11 @@
+package daemon
```

Comment:
No, I asked about `_windows.go` exactly. Maybe it was @ahmetalpbalkan who said this. Anyway I can find tons files with tags and postfixes.
I little bikeshedding now because I asked to delete those tags, but it was said that that is "convention" of microsoft or something like this.